models/credmark/algorithms/portfolio/sharpe.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+from credmark.cmf.model import Model
+from credmark.dto import DTO, DTOField
+from credmark.cmf.types import Token, Price
+
+import numpy as np
+import pandas as pd
+
+
+class SharpRatioInput(DTO):
+    token: Token
+    return_rolling_days: int = DTOField(182, ge=2)
```

Comment:
interesting to hardcode in the "days" unit. I guess I'm ok with it, but I don't LOVE it ;) could we do a generic dto for interval-based agg stuff?

Reply:
Ok. got you. I have updated the model input to use generic window. The rolling return can be derived as half the window.

```
credmark-dev run finance.sharpe-ratio-token --input '{"token": {"address": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9"}, "window": "10 minute", "risk_free_rate": 0.02}' -b 14234904 --format_json
```

Or,
```
credmark-dev run finance.sharpe-ratio-token --input '{"token": {"address": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9"}, "window": "360 days", "risk_free_rate": 0.02}' -b 14234904 --format_json
```